backend/src/test/java/com/google/sps/TestEmailNotifier.java

```diff
@@ -0,0 +1,109 @@
+package com.google.sps;
+
+import com.google.sps.notifs.EmailNotifier;
+import org.junit.Assert;
+import org.junit.Test;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+import java.io.IOException;
+import java.util.Properties;
+import static org.junit.Assert.*;
+
+public class TestEmailNotifier {
```

Comment:
tiny nit: the file and class name should be EmailNotifierTest. The convention for test classes is to add "Test" on the end of the unit being tested. Confusingly, by convention a TestFoo is a version of the Foo class that's meant as a helper for testing.

Reply:
Done !